source/localizable/applications/services.md

```diff
@@ -63,7 +63,7 @@ To access a service,
 you can inject it in any container-resolved object such as a component or another service using the `Ember.inject.service` function.
 There are 2 ways to use this function.
 You can either invoke it with no arguments, or you can pass it the registered name of the service.
-When no arguments are passed the services is loaded based in the name of the variable key.
```

Comment:
Please change the text only in this version and leave the others untouched, CrowdIn will synchronize properly :)